Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM node:alpine
+FROM andthensome/alpine-node-bash
```

Comment:
Deis requires you to have an image that:
1. uses the `EXPOSE` directive to expose only one port
2. that port must be listening for an HTTP connection
3. the image must use the `CMD` directive to define the entry point
4. the image **must** contain `bash`